tasks/tests/test_view_single_task.py

```diff
@@ -69,3 +71,17 @@ def test_add_new_comment(self, client, task_1, employee_1, content, difference):
             assert last_comment.description == content
             assert last_comment.appUser == employee_1
             assert last_comment.task == task_1
+
+    @pytest.mark.parametrize('comment_len',
```

Comment:
IMHO you can add these tests into the previous parametrize test like:
     @pytest.mark.parametrize('content, difference',
                             [
                                ('This is a test comment', 1),
                                (string.punctuation, 1),
                                (string.ascii_letters + string.digits, 1),
                                ('', 0),
                                ('a'*(MAX_LEN+1), 0),
                             ],
                             ids=[
                                "valid_comment_success",
                                "special_chars_success",
                                "chars_and_digits_success",
                                "empty_comment_fails",
                                "long_comment_fails",
                             ]
                             )
    def test_add_new_comment(self, client, task_1, employee_1, content, difference):

Reply:
Done.